ci/runtests.ps1

```
@@ -0,0 +1,4 @@
+.\venv\Scripts\activate.ps1
+coverage run -m behave .\tests\features
+if ($LastExitCode -ne 0) { throw }
+coveralls
```

worth noting this can fail silently